store/snapshots/manager.go

```diff
@@ -550,3 +550,9 @@
 		m.logger.Debug("pruned state snapshots", "pruned", pruned)
 	}
 }
+
+// close snapshot database
+func (m *Manager) Close() {
```

Comment:
Should we call this in BaseApp.Close?

Reply:
Called in next PR 
https://github.com/cosmos/cosmos-sdk/pull/17094/files#diff-328a55b5ac5816102544792c5c6a75df50df3b9b01afe770c9abc2da79b3e6afR1080